DEPLOYMENT_CHECKLIST.md

```diff
@@ -1,108 +1,74 @@
 # Deployment Checklist
 
-This checklist is seen as a guide to deploy the stack to a new chain.
+This checklist is seen as a guide to deploy the contracts to a new chain.
 
 ## Pre-Deployment
 
+- [ ] Run `yarn` in the repository root to install the dependencies.
+- [ ] Run `yarn build` in `packages/contracts` to make sure the contracts compile.
+- [ ] Run `yarn test` in `packages/contracts` to make sure the contract tests succeed.
+  - In order to run the tests on localhost, add the following in the packages/contracts/.env
+    ```
+    HARDHAT_DAO_ENS_DOMAIN="testdao.eth"
+    HARDHAT_PLUGIN_ENS_DOMAIN="testpluginrepo.eth"
+    ```
 - [ ] Verify that the deployers wallet has enough funds.
-- [ ] Check that the subgraph hoster supports the network OSx is deployed to.
-- [ ] Make sure you are using Node v16
```

Comment:
Isn't this important info for them? If you do not use Node v16, you experience errors

Reply:
Well, as soon as the script is run, they will immediatelly note this and the script works for node version > 16. It's highly unlikely that anyone still uses less than 16 version. So all good IMO.